scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+  default value: 'false'
+  enumerated possible values: true,false
+  Description: Operate in base station mode
+  Notes: 'This flag ultimately determines whether the SBP message with identifier
+    MSG_BASE_POS will be calculated and sent.  Logically, setting this attribute to
+    "true" sets the Local Piksi as a base station and configures the unit to send
+    its surveyed location coordinates to the other Piksi(s) with which the base
+    station is communicating.  If "true", the Piksi that receives the base station''s
+    surveyed coordinates will also calculate and communicate its own pseudo absolute
+    RTK position based upon the received position of the Base Station.  '
+- group: base_station_mode
+  name: surveyed lat
+  type: Double
+  units: degrees
+  default value: '0'
+  enumerated possible values:
+  Description: Surveyed latitude of the Piksi connected to the console over usb
+  Notes: This setting represents the latitude of the connected Piksi's antenna.  If
+    "base station mode" is "true", the coordinate will be communicated to the Rover
+    with which to calculate the Rover's pseudo-absolute RTK position.  The value should
+    be as accurate as possible and should have precision to at least 7 digits following
+    the decimal point.  For reference, 1e-7 degrees of latitude is about 1.1cm on
+    the surface of the earth. Any errors in the surveyed position will directly affect
+    the pseudo-absolute RTK position measurement reported by the Rover.
+- group: base_station_mode
+  name: surveyed lon
+  type: Double
+  units: degrees
+  default value: '0'
+  enumerated possible values:
+  Description: Surveyed longitude of the Piksi connected to the console over usb
+  Notes: This setting represents the longitude of the connected Piksi's antenna.  If
+    "base station mode" is "true", the coordinate will be communicated to the Rover
+    unit with which to calculate the Rover's pseudo-absolute RTK position.  This value
+    should be as accurate as possible and should have precision to at least 7 digits
+    following the decimal point. For reference, 1e-7 degrees of longitude at 35 degrees
+    latitude is about 1 cm. Any errors in the surveyed position will directly affect
+    the pseudo-absolute RTK position measurement reported by the Rover.
+- group: base_station_mode
+  name: surveyed alt
+  type: Double
+  units: meters
+  default value: '0'
+  enumerated possible values:
+  Description: Surveyed altitude of the Piksi connected to the console over usb
+  Notes: This setting represents the altitude of the antenna connected to the Piksi
+    above the WGS84 ellipsoid. This coordinate will be communicated to the Rover
+    against which to calculate a pseudo-absolute position. This value should be as
+    accurate as possible and should be precise to 1 cm.  Any errors in the surveyed
+    position will directly affect the pseudo-absolute RTK position measurement reported
+    by the Rover.
+- group: float_kf
```

Comment:
Agree, I'll make this a separate enhancement pull request.  It will also have two tabs, "basic" and "advanced".  We will hide the obscure things in advanced.